src/by/sergey_l/lesson3/Task6.java

```diff
@@ -0,0 +1,15 @@
+package by.sergey_l.lesson3;
+
+/*
+Напишите программу вывода всех четных чисел от 2 до 100 включительно
+
+ */
+
+public class Task6 {
+
+    public static void main(String[] args) {
+        for (int i = 2; i <= 100; i += 2) {
+            System.out.println(i);
+        }
+        }
+    }
```

Comment:
use code formatting, correct indentation

Reply:
I fixed it